Helper/Price.php

```diff
@@ -203,7 +221,21 @@ public function getProductPrice( // @codingStandardsIgnoreLine
                 break;
 
             default:
-                $price = $finalPrice ? $product->getFinalPrice() : $product->getPrice();
+                $date = $this->localeDate->scopeDate();
+                $wid = $product->getStore()->getWebsiteId();
+                $gid = 0;
```

Comment:
Should we use this one? GroupManagement::NOT_LOGGED_IN_ID;
Or get the default group from: GroupManagement::XML_PATH_DEFAULT_ID;

Reply:
`GroupManagement::XML_PATH_DEFAULT_ID` seems to be the default group id for new user when a user registers into a store. That's why we must go `GroupManagement::NOT_LOGGED_IN_ID`